ports/nrf/boards/SEEED_XIAO/deploy.md

```diff
@@ -0,0 +1,50 @@
+# Make and install a firmware package for SEEED nrf52840:
+
+1. Install adafruit-nrfutil with `apt-get install adafruit-nrfutil`
+   The official NRF nrfutil suite used in Makefile did not work for me and
+   failed consistently with Python errors. Maybe a wrong Python version,
+   but obviously it misses version checks.
```

Comment:
Please reword this to something like "Install the nrfutil suite.  If this does not work then you can try adafruit-nrfutil ..." (better not to use words like "I" and "me" in docs).